### PR TITLE
translation: Improvements and fixes in French translation

### PR DESCRIPTION
**Pull request by aginies**
Few improvments fixes and one error fix in the French po file.

### CODE REVIEW

**Comment by bkerler** [comment]
Hi, one translation is wrong : Buddy Board. The board itself is called "Buddy", thus should not be translated.

File: src/lang/po/fr/Prusa-Firmware-Buddy_fr.po [hunk 1]
```diff
@@ -326,7 +326,7 @@ msgstr ""
 "Version du Bootloader\n"
 "%d.%d.%d\n"
 "\n"
-"Buddy Board\n"
```
Comment (bkerler) [inline src/lang/po/fr/Prusa-Firmware-Buddy_fr.po:329]: This should be "Buddy Board" as it refers to the xBuddy board, thus shouldn't be translated.

File: src/lang/po/fr/Prusa-Firmware-Buddy_fr.po [hunk 2]
```diff
@@ -3542,7 +3542,7 @@ msgstr "Aucune configuration"
 
 #: src/gui/MItem_tools.cpp:287
 msgid "No crash dump to save."
-msgstr "Aucun dump de crash à sauvegarder."
+msgstr "Aucun rapport de vidage de mémoire à sauvegarder."
```
Comment (bkerler) [inline src/lang/po/fr/Prusa-Firmware-Buddy_fr.po:3545]: A "rapport" is a report, in this case, it is a crash dump, thus the translation is correct.

**Comment by bkerler** [comment]
Also please relabel your PR to "translation: Improvements and fixes in French translation" according to the Contribution guidelines [here](https://github.com/prusa3d/Prusa-Firmware-Buddy/blob/master/doc/contributing.md). Thanks !

File: src/lang/po/fr/Prusa-Firmware-Buddy_fr.po (hunk 1 above)
Comment (aginies) [reply src/lang/po/fr/Prusa-Firmware-Buddy_fr.po]: fixed

File: src/lang/po/fr/Prusa-Firmware-Buddy_fr.po (hunk 2 above)
Comment (aginies) [reply src/lang/po/fr/Prusa-Firmware-Buddy_fr.po]: fixed (reverted)

**Comment by vorner** [comment]
Internal reference: BFW-5039

File: src/lang/po/fr/Prusa-Firmware-Buddy_fr.po [hunk 3]
```diff
@@ -76,7 +76,7 @@ msgstr "Acquisition du code d'enregistrement, veuillez patienter..."
 #. window_menu
 #: src/gui/MItem_tools.cpp:285
 msgid "A crash dump is being saved."
-msgstr "Un dump sur crash est en cours d'enregistrement."
+msgstr "Un rapport de vidage de mémoire est en cours d'enregistrement."
```
Comment (teotwaki) [inline src/lang/po/fr/Prusa-Firmware-Buddy_fr.po:79]: This is not a correct translation. "rapport de vidage de mémoire" would translate to "a memory emptying report". A "crash dump" is the opposite: it's a dump of the microcontroller's (or process') memory relevant to the crash.

What would you think of using "Une copie de la mémoire vive est en cours d'enregistrement."

File: src/lang/po/fr/Prusa-Firmware-Buddy_fr.po (hunk 1 above)
Comment (teotwaki) [reply src/lang/po/fr/Prusa-Firmware-Buddy_fr.po]: This does not appear fixed

File: src/lang/po/fr/Prusa-Firmware-Buddy_fr.po (hunk 2 above)
Comment (teotwaki) [reply src/lang/po/fr/Prusa-Firmware-Buddy_fr.po]: Please see my other comment; crash dump is not related to "vidage de mémoire".

**Comment by teotwaki** [comment]
If it is still relevant, could you also include the "déchargé" mentioned here: https://github.com/prusa3d/Prusa-Firmware-Buddy/issues/3372

File: src/lang/po/fr/Prusa-Firmware-Buddy_fr.po (hunk 3 above)
Comment (aginies) [reply src/lang/po/fr/Prusa-Firmware-Buddy_fr.po]: ok, i will include it.

**Comment by michalrudolf** [comment]
@aginies Your improvements to our french translations are appreciated, we just need to neaten it a bit.
- All of the commits, that edit french .po file, should be squashed into one commit (you can list the various changes in the commit message).
- You have to rebase the changes on current prusa3d:master.

You can try to rebase in place with rebase --onto, but since you commited all the changes into your fork's master, it may be a little difficult.

I would suggest you close this PR and create a new one, that will merge from new_branch to prusa3d:master:
1) Do a soft reset of all of your commits `git reset XXXX` where XXXX is hash of the last master's commit before your changes
2) Stash the changes `git stash`
3) Pull current prusa3d::master `git pull {prusa3d remote} master`
4) Create a new branch on up-to-date master `git checkout -b {new_branch name}`
5) Pop the stashed changes `git stash pop`
6) Commit all the changes
7) Push the branch to your fork and create a new PR
8) Mention me and this PR in the new PR's description

If you have any trouble, just let me know.

**Comment by michalrudolf** [comment]
To speed up the process, I have rebased it on the current master here https://github.com/prusa3d/Prusa-Firmware-Buddy/pull/4016. Thank you for the contribution.

**Pull request closed without merging**